public/hwclayer.h

```diff
@@ -18,8 +18,10 @@
 #define PUBLIC_HWCLAYER_H_
 
 #include <hwcdefs.h>
-
 #include <platformdefines.h>
+#include "hdr_metadata_defs.h"
```

Comment:
hdr_metadata_defs.h is also in public folder as same as hwcdefs.h
it is better to use <hdr_metadata_defs.h>